install/kubernetes/cilium/templates/cilium-agent/daemonset.yaml

```diff
@@ -454,6 +455,49 @@ spec:
               - SYS_PTRACE
           {{- end}}
       {{- end }}
+      - name: apply-sysctl-overwrites
+        image: {{ include "cilium.image" .Values.image | quote }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        env:
+        - name: BIN_PATH
+          value: {{ .Values.cni.binPath }}
+        command:
+        - sh
+        - -ec
+        # The statically linked Go program binary is invoked to avoid any
+        # dependency on utilities like sh that can be missing on certain
+        # distros installed on the underlying host. Copy the binary to the
+        # same directory where we install cilium cni plugin so that exec permissions
+        # are available.
+        - |
+          cp /usr/bin/cilium-sysctlfix /hostbin/cilium-sysctlfix;
+          nsenter --mount=/hostproc/1/ns/mnt "${BIN_PATH}/cilium-sysctlfix";
+          rm /hostbin/cilium-sysctlfix
```

Comment:
Does there need to be a semi-colon at the end of these lines?

Reply:
Not sure to be honest :sweat_smile:, I copied this part from the `cilium-mount` container at line 429, I haven't tried it without to see how it gets interpreted